src/Repository/AccessTokenRepository.php

```diff
@@ -58,6 +58,19 @@ public function findToken(string $token): ?IAccessToken
         return null;
     }
 
+    public function deleteTokensByAbility(string $ability): void
+    {
+        $expr = $this->em->getExpressionBuilder();
+
+        $this->em
+            ->createQueryBuilder()
+            ->delete($this->tokenModel, 'ac')
+            ->where($expr->like('ac.abilities', ':ability'))
```

Comment:
Is er een reden waarom je kiest voor de `like` ipv IN? Hier zouden scenarios kunnen optreden welke niet gewenst zijn als een ability toevallig een gedeelte van een andere ability bevat en je deze wilt verwijderen.

Reply:
`in` werkt niet voor array fields. 

Quotes toegevoegd aan de query parameter voor extra veiligheid.